src/OrdinaryDiffEq.jl

```diff
@@ -410,7 +410,8 @@ export constructDormandPrince
 
 export FunctionMap, Euler, Heun, Ralston, Midpoint, RK4, ExplicitRK, OwrenZen3, OwrenZen4,
        OwrenZen5,
-       BS3, BS5, DP5, Tsit5, DP8, TanYam7, TsitPap8, CompositeAlgorithm, Anas5, RKO65, FRK65, PFRK87,
+       BS3, BS5, DP5, Tsit5, DP8, TanYam7, TsitPap8, CompositeAlgorithm, Anas5, RKO65,
+       FRK65, PFRK87,
```

Comment:
are you sure you're on the right formatter version?

Reply:
I am using this version of JuliaFormatter

[98e50ef6] JuliaFormatter v1.0.56